src/components/TransactionsTab/TransactionsTab.tsx

```diff
@@ -109,86 +129,121 @@ const TransactionsTab = ({
-            <WalletLink chainId={chainId!} address={address!} />
-          </WalletLinkContainer>
-        </WalletHeader>
-
-        <TransactionsContainer ref={transactionsScrollRef} $overflow={overflow}>
-          <Legend>
-            <LegendLine>{t("wallet:activeTransactions")}</LegendLine>
-          </Legend>
-          <TransactionContainer>
-            {pendingTransactions.length ? (
-              pendingTransactions.map((transaction) => (
-                <WalletTransaction
-                  transaction={transaction}
-                  tokens={tokens}
-                  chainId={chainId!}
-                  key={`${transaction.hash}-${transaction.nonce}`}
-                />
-              ))
-            ) : (
-              <NoTransactions>
-                <IconContainer>
-                  <Icon name="transaction" />
-                </IconContainer>
-                {t("wallet:noActiveTransactions")}
-              </NoTransactions>
-            )}
-          </TransactionContainer>
-          {completedTransactions && (
+    <AnimatePresence>
+      {open && (
+        <Container
+          ref={containerRef}
+          animate={{ x: 0 }}
+          transition={{ duration: 0.3 }}
+          initial={{ x: "24rem" }}
+          exit={{ x: "24rem" }}
+        >
+          <BackButton
+            animate={{ y: 0 }}
+            transition={{ duration: 0.5 }}
+            initial={{ y: "-5rem" }}
+            exit={{ opacity: 0 }}
+            onClick={() => setTransactionsTabOpen(false)}
+          >
+            <Icon name="chevron-right" iconSize={1} />
+          </BackButton>
+          <WalletHeader>
+            <NetworkInfoContainer>
+              <NetworkName>
+                {addressMapping[chainId] || t("wallet:unsupported")}
+              </NetworkName>
+              <Balances>{formatUnits(balance).substring(0, 5)} ETH</Balances>
+            </NetworkInfoContainer>
+            <WalletAnchorTag
+              target="_blank"
+              rel="noreferrer"
+              href={
+                chainId
+                  ? `https://${hrefAddressMapping[chainId]}etherscan.io/address/${address}`
+                  : ""
+              }
+            >
+              <BorderedButton>
+                <ConnectionStatusCircle $connected={!!address} />
+                <InfoHeading>
+                  {isUnsupportedNetwork
+                    ? t("wallet:unsupported")
+                    : addressOrName
+                    ? addressOrName
+                    : t("wallet:notConnected")}
+                </InfoHeading>
+              </BorderedButton>
+            </WalletAnchorTag>
+          </WalletHeader>
+
+          <TransactionsContainer
+            ref={transactionsScrollRef}
+            $overflow={overflow}
+          >
             <Legend>
-              <LegendLine>{t("wallet:completedTransactions")}</LegendLine>
+              <LegendLine>{t("wallet:activeTransactions")}</LegendLine>
```

Comment:
Fix the translation for activeTransactions and completedTransactions to "Active" and "Completed", and use style `text-transform: uppercase` in css.